### PR TITLE
 fix: add ais-RangeInput--noRefinement CSS class on root when min === max

### PR DESCRIPTION
**Pull request by tkrugg**
**Summary**

This adds `ais-RangeInput--noRefinement` CSS class on root when min === max.
This is specified in https://instantsearch-css.netlify.com/widgets/range-input/

Currently every flavour is doing it its won way:

[InstantSearch.JS](https://github.com/algolia/instantsearch.js/blob/b6ee2596d353ea692d91929759bc2b2fe34945c7/src/components/RangeInput/RangeInput.js#L40) is the following condition: `Boolean(min || max)`.

[React InstantSearch](https://github.com/algolia/react-instantsearch/blob/master/packages/react-instantsearch-core/src/connectors/connectRange.js#L247) is using facet count to enable this CSS class but this is not exposed in InstantSearch Connector.

[Vue InstantSearch](https://github.com/algolia/vue-instantsearch/blob/ceeecefa9948526f0518a00eb68563298fcdb7bb/src/components/RangeInput.vue#L62) enables it when `!canRefine`, where `canRefine = state.range && state.range.min !== state.range.max`.

This PR uses the same condition as Vue InstantSearch.

### CODE REVIEW

**Comment by eunjae-lee** [comment]
> would be best to move these conditions inside InstantSearch, wdyt?

I agree. Hopefully all that kind of conditions could be provided by connectors inside InstantSearch.js.

**Comment by francoischalifour** [comment]
@Haroenv That's what we should do indeed. The downside is that it can be considered as a breaking change in InstantSearch.js so it would be part of v4. I'm not sure we want to wait for InstantSearch.js v4 before releasing Angular InstantSearch v3 stable.

Since our behaviors regarding `noRefinement` are inconsistent anyway, perhaps we should implement the ones we think make sense in Angular and replicate them for InstantSearch.js v3?

File: src/range-input/range-input.ts [hunk 1]
```diff
@@ -87,6 +90,10 @@ export class NgAisRangeInput extends BaseWidget {
     return 1 / Math.pow(10, precision);
   }
 
+  get canRefine() {
+    return this.state.range && this.state.range.min !== this.state.range.max;
```
Comment (francoischalifour) [inline src/range-input/range-input.ts]: According to [the typings](https://github.com/algolia/angular-instantsearch/blob/380cf9bcd175922da41293a2d2c9d450188dd7ad/src/range-input/range-input.ts#L10), `this.state.range` is always defined. Is the first check necessary?

Comment (tkrugg) [reply src/range-input/range-input.ts]: solved here 89e1113

**Pull request merged**